app/build.gradle

```diff
@@ -243,6 +243,7 @@ dependencies {
     compile "io.reactivex.rxjava2:rxjava:${RX_JAVA_VERSION}"
     //compile 'net.ltgt.dagger:bullet:1.0-SNAPSHOT' // Loaded from JitPack
     compile "org.parceler:parceler-api:${PARCELER_VERSION}"
+    compile files('libs/YouTubeAndroidPlayerApi.jar')
```

Comment:
Was this the only way to include the dependency? Also consider that this line is not necessary because you have the one below.